### ISSUE TITLE
Bug: ipvanish openvpn zip file url needs to be updated

### ISSUE THREAD

**Issue by Nsm564** [qdm12/gluetun#2556]
### Is this urgent?

Yes

### Host OS

ubuntu 24

### CPU arch

aarch64

### VPN service provider

IPVanish

### What are you using to run the container

Portainer

### What is the version of Gluetun

latest tag 1/11/2024

### What's the problem 🤔

gluetun isnt working with ipvanish anymore or for me atleast im sure the user/password correct. 

idk how this stuff works but i checked ipvanish website u can download openvpn config zip file it was updated like 2 days ago. just a hint if it helps. ty

### Share your logs (at least 10 lines)

```plain text
2024-11-01T03:57:36Z INFO [firewall] allowing VPN connection...

2024-11-01T03:57:36Z INFO [openvpn] OpenVPN 2.6.11 aarch64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]

2024-11-01T03:57:36Z INFO [openvpn] library versions: OpenSSL 3.3.2 3 Sep 2024, LZO 2.10

2024-11-01T03:57:36Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.88.121:443

2024-11-01T03:57:36Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:57:36Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.88.121:443

2024-11-01T03:58:23Z INFO [healthcheck] program has been unhealthy for 46s: restarting VPN

2024-11-01T03:58:23Z INFO [healthcheck] 👉 See https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md

2024-11-01T03:58:23Z INFO [healthcheck] DO NOT OPEN AN ISSUE UNLESS YOU READ AND TRIED EACH POSSIBLE SOLUTION

2024-11-01T03:58:23Z INFO [vpn] stopping

2024-11-01T03:58:23Z INFO [vpn] starting

2024-11-01T03:58:23Z INFO [firewall] allowing VPN connection...

2024-11-01T03:58:23Z INFO [openvpn] OpenVPN 2.6.11 aarch64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]

2024-11-01T03:58:23Z INFO [openvpn] library versions: OpenSSL 3.3.2 3 Sep 2024, LZO 2.10

2024-11-01T03:58:23Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.112.8:443

2024-11-01T03:58:23Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:58:23Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.112.8:443

2024-11-01T03:58:24Z INFO [openvpn] [ams-c02.ipvanish.com] Peer Connection Initiated with [AF_INET]216.131.112.8:443

2024-11-01T03:58:24Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong 🤨

2024-11-01T03:58:24Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2024-11-01T03:58:34Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.112.8:443

2024-11-01T03:58:34Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:58:34Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.112.8:443

2024-11-01T03:58:35Z INFO [openvpn] [ams-c02.ipvanish.com] Peer Connection Initiated with [AF_INET]216.131.112.8:443

2024-11-01T03:58:35Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong 🤨

2024-11-01T03:58:35Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2024-11-01T03:58:45Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.112.8:443

2024-11-01T03:58:45Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:58:45Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.112.8:443

2024-11-01T03:58:46Z INFO [openvpn] [ams-c02.ipvanish.com] Peer Connection Initiated with [AF_INET]216.131.112.8:443

2024-11-01T03:58:46Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong 🤨

2024-11-01T03:58:46Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2024-11-01T03:58:56Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.112.8:443

2024-11-01T03:58:56Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:58:56Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.112.8:443

2024-11-01T03:58:57Z INFO [openvpn] [ams-c02.ipvanish.com] Peer Connection Initiated with [AF_INET]216.131.112.8:443

2024-11-01T03:58:57Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong 🤨

2024-11-01T03:58:57Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2024-11-01T03:59:07Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.112.8:443

2024-11-01T03:59:07Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:59:07Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.112.8:443

2024-11-01T03:59:08Z INFO [openvpn] [ams-c02.ipvanish.com] Peer Connection Initiated with [AF_INET]216.131.112.8:443

2024-11-01T03:59:08Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong 🤨

2024-11-01T03:59:08Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2024-11-01T03:59:14Z INFO [healthcheck] program has been unhealthy for 51s: restarting VPN

2024-11-01T03:59:14Z INFO [healthcheck] 👉 See https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md

2024-11-01T03:59:14Z INFO [healthcheck] DO NOT OPEN AN ISSUE UNLESS YOU READ AND TRIED EACH POSSIBLE SOLUTION

2024-11-01T03:59:14Z INFO [vpn] stopping

2024-11-01T03:59:14Z INFO [vpn] starting

2024-11-01T03:59:14Z INFO [firewall] allowing VPN connection...

2024-11-01T03:59:14Z INFO [openvpn] OpenVPN 2.6.11 aarch64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]

2024-11-01T03:59:14Z INFO [openvpn] library versions: OpenSSL 3.3.2 3 Sep 2024, LZO 2.10

2024-11-01T03:59:14Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.88.139:443

2024-11-01T03:59:14Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:59:14Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.88.139:443

2024-11-01T03:59:14Z INFO [openvpn] VERIFY ERROR: depth=1, error=self-signed certificate in certificate chain: C=US, ST=VPN, L=VPN, O=VPN, OU=VPN, CN=VPN, name=VPN, emailAddress=VPN, serial=11865505408368628294

2024-11-01T03:59:14Z INFO [openvpn] OpenSSL: error:0A000086:SSL routines::certificate verify failed:

2024-11-01T03:59:14Z INFO [openvpn] TLS_ERROR: BIO read tls_read_plaintext error

2024-11-01T03:59:14Z INFO [openvpn] TLS Error: TLS object -> incoming plaintext read error

2024-11-01T03:59:14Z INFO [openvpn] TLS Error: TLS handshake failed

2024-11-01T03:59:14Z INFO [openvpn] SIGTERM received, sending exit notification to peer

2024-11-01T03:59:14Z INFO [openvpn] SIGTERM[soft,tls-error] received, process exiting

2024-11-01T03:59:14Z INFO [vpn] retrying in 15s

2024-11-01T03:59:29Z INFO [firewall] allowing VPN connection...

2024-11-01T03:59:29Z INFO [openvpn] OpenVPN 2.6.11 aarch64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]

2024-11-01T03:59:29Z INFO [openvpn] library versions: OpenSSL 3.3.2 3 Sep 2024, LZO 2.10

2024-11-01T03:59:29Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.88.249:443

2024-11-01T03:59:29Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T03:59:29Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.88.249:443

2024-11-01T04:00:10Z INFO [healthcheck] program has been unhealthy for 56s: restarting VPN

2024-11-01T04:00:10Z INFO [healthcheck] 👉 See https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md

2024-11-01T04:00:10Z INFO [healthcheck] DO NOT OPEN AN ISSUE UNLESS YOU READ AND TRIED EACH POSSIBLE SOLUTION

2024-11-01T04:00:10Z INFO [vpn] stopping

2024-11-01T04:00:10Z INFO [vpn] starting

2024-11-01T04:00:10Z INFO [firewall] allowing VPN connection...

2024-11-01T04:00:10Z INFO [openvpn] OpenVPN 2.6.11 aarch64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD]

2024-11-01T04:00:10Z INFO [openvpn] library versions: OpenSSL 3.3.2 3 Sep 2024, LZO 2.10

2024-11-01T04:00:10Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]216.131.88.21:443

2024-11-01T04:00:10Z INFO [openvpn] UDPv4 link local: (not bound)

2024-11-01T04:00:10Z INFO [openvpn] UDPv4 link remote: [AF_INET]216.131.88.21:443
```

### Share your configuration

```yml
services:  

  gluetun:
    image: qmcgaw/gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks

    volumes:
      - /docker/appdata/gluetun:/gluetun
    environment:     
      - VPN_SERVICE_PROVIDER=ipvanish
      - OPENVPN_USER=***
      - OPENVPN_PASSWORD=***
      - SERVER_COUNTRIES=Netherlands
```

**Comment by Nsm564** [comment]
i tried updating servers data, still same problem i dont have other vpn subsciption to try but i think its only ipvanish that is not working.

**Comment by IPVanish-Support** [comment]
Hi Guys, We don't know if this will help your project, but we have recently updated our OpenVPN configs to support OpenVPN 2.6 and above - [IPVanish configs OpenVPN 2.6+](https://configs.ipvanish.com/openvpn/v2.6.0-0/)

**Comment by qdm12** [comment]
Thanks @IPVanish-Support for your help 💯 !

Zip file URL updated in 5f182febae6ba3cb2941a4c602d2af871befc775 and built-in servers data also updated in ad6c52dc4c15a417a48d0ce7e57b4c0b8ba16341 (latest image and future v3.40.0 release)

@IPVanish-Support A few extra questions if you don't mind:

1. Are all your servers supporting both Openvpn 2.5 and 2.6, or only 2.6?
2. Are your server hostnames for Openvpn 2.5 and 2.6 the same?

Thanks!

**Comment by IPVanish-Support** [comment]
You're welcome 👍  See our answers to your questions below:

> A few extra questions if you don't mind:
> 
> 1. Are all your servers supporting both Openvpn 2.5 and 2.6, or only 2.6? **Yes for now.**
> 2. Are your server hostnames for Openvpn 2.5 and 2.6 the same? **Yes**

**Comment by Nsm564** [comment]
still not working for me, idk might be something on my end.

**Comment by qdm12** [comment]
Thanks @IPVanish-Support

@Nsm564 try the latest image again (pull it 😉) - 94dfb2b1f2a000f9ee19d300ec4850fbe3e49cda updates the openvpn configurations regarding the CA value and adds the compression option `comp-lzo` which seems to now be present in all the openvpn configuration files of ipvanish.

**Comment by qdm12** [comment]
Closing this assuming it's resolved. If not please create another issue referencing this one! Thanks!
